packages/markdown-docx/src/constants.js

```diff
@@ -25,13 +25,16 @@ const TRANSFORMED_NODES = {
     emphasize: `${NS_PREFIX_CommonMarkModel}Emph`,
     heading: `${NS_PREFIX_CommonMarkModel}Heading`,
     item: `${NS_PREFIX_CommonMarkModel}Item`,
+    link: `${NS_PREFIX_CommonMarkModel}Link`,
     paragraph: `${NS_PREFIX_CommonMarkModel}Paragraph`,
     softbreak: `${NS_PREFIX_CommonMarkModel}Softbreak`,
     strong: `${NS_PREFIX_CommonMarkModel}Strong`,
     text: `${NS_PREFIX_CommonMarkModel}Text`,
     thematicBreak: `${NS_PREFIX_CommonMarkModel}ThematicBreak`,
     variable: `${NS_PREFIX_CiceroMarkModel}Variable`,
-    clause:`${NS_PREFIX_CiceroMarkModel}Clause`
+    clause: `${NS_PREFIX_CiceroMarkModel}Clause`,
 };
 
-module.exports = { TRANSFORMED_NODES };
+const RELATIONSHIP_OFFSET = 2;
```

Comment:
Please add comments because I don't think so anyone can infer what this offset is for. The best idea would be to keep this in the `helpers.js` file only but it can't go there semantically.

Reply:
@algomaster99 
I would prefer keeping it in constants and adding suitable comments here itself.